internal/version.go

```diff
@@ -30,7 +30,7 @@ package internal
 const (
 	// SDKVersion is a semver (https://semver.org/) that represents the version of this Temporal GoSDK.
 	// Server validates if SDKVersion fits its supported range and rejects request if it doesn't.
-	SDKVersion = "1.25.1"
+	SDKVersion = "1.26.0"
```

Comment:
Note I did the full release here, possible we should put `-rc.1` that implies we trust what parses this to follow semvar

Reply:
I think what you have here now is ok too.